File: 58-LengthOfLastWord/58-LengthOfLastWord.java
// Last updated: 11/11/2025, 1:57:55 AM
class Solution {
    public int lengthOfLastWord(String s) {
        String tempString = s.trim();
        int index = tempString.lastIndexOf(" ");
        if (index == 0) {
            return (s.length());
        }
        tempString = tempString.substring(index+1); //it counts the index as part of the substring
        return (tempString.length());
    }
} 

File: 58-LengthOfLastWord/58-LengthOfLastWord.py
# Last updated: 11/11/2025, 1:57:53 AM
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        Count last word
        """ 

        length = 0
        i = len(s) - 1

        while i >= 0 and s[i] == " ":
            i -= 1

        while i >= 0 and s[i] != " ":
            length += 1
            i -= 1

        return length
        
                

                
                
        return len(seen)


        